TheJumpCloud/support
# remove existing desktop symlink when removing app

## Issues
* [<jira-id>](https://jumpcloud.atlassian.net/browse/<jira-id>) - <jira-title>

## What does this solve?
The desktop symlink gets broken when running the old script multiple times
An error shows indicating that the app is damaged and can't be opened.
The app is not actually damaged and can be opened from ~/Applications  (right click -> open -> open)
The updated script removes the old symlink and the recreated one will work as intended

## Is there anything particularly tricky?
No

## How should this be tested?
Run it multiple times and verify that the app launches normally without error messages

## Screenshots
